spec/support/aruba.rb

```diff
@@ -15,10 +16,8 @@ def dirs
 end
 
 RSpec.configure do |c|
-  c.include SpecHelper::Aruba
-
   c.before :each do
-    clean_current_dir
     restore_env
+    setup_aruba
```

Comment:
There's no need to use RSpec.configure here. You can safely remove the whole block. `restore_env` and  `setup_aruba` are run via `aruba/rspec`.